services/github/src/constants.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  CONCLUSIONS: { // maps status communicated by the queue service to github checkrun conclusions
+    /*eslint quote-props: ["error", "consistent-as-needed"]*/
+    'completed': 'success',
+    'failed': 'failure',
+    'exception': 'failure',
+    'deadline-exceeded': 'timed_out',
+    'canceled': 'cancelled',
+    'superseded': 'neutral', // queue status means: is not relevant anymore
+    'claim-expired': 'failure',
+    'worker-shutdown': 'neutral', // queue status means: will be retried
+    'malformed-payload': 'action_required', // github status means "correct your task definition"
+    'resource-unavailable': 'failure',
+    'internal-error': 'failure',
+    'intermittent-task': 'neutral', // queue status means: will be retried
+  },
+  CHECKRUN_TEXT: 'View task in Taskcluster',
+  CUSTOM_CHECKRUN_TEXT_ARTIFACT_NAME: 'customCheckRunText.md',
```

Comment:
I'm guessing this probably should have a path associated to it (e.g. `public/gituhb/`)?

Reply:
public by default?

Reply:
I think so, yes. I don't think workers support private logs currently, and I expect this to be used mostly for summarizing stuff from them, so having it be public seems fine. I also think all the repos actively used are public (or could be), so the checks will be visible anyway. (As I said elsewhere, I think the initial implementation should only support public artifacts)

Reply:
Gotcha. Like I mentioned above, your suggestion on supporting public artifacts only made sense to me (I made sure with the Fenix folks - they confirmed it's fine for their needs), so agreed.